crates/astria-sequencer/src/app/storage/values/revision_number.rs

```diff
@@ -0,0 +1,56 @@
+use std::fmt::{
+    self,
+    Display,
+    Formatter,
+};
+
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+
+use super::{
+    Value,
+    ValueImpl,
+};
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(in crate::app) struct RevisionNumber(u64);
```

Comment:
Also should have a domain type for this one.

Reply:
Ticketed at #1599.